nexus/db-queries/src/db/datastore/inventory.rs

```diff
@@ -222,6 +213,23 @@ impl DataStore {
             })
             .collect();
 
+        // Build up a list of `OmicronSledConfig`s we need to insert (each sled
+        // has 0-3), all of their zones, zone NICs, datasets, and disks, and
```

Comment:
0-3 sled configs? That sounds strange to me, I'd expect there to be exactly 1.

edit: ahh this is explained below. Worth adding a comment pointing to that explanation.

Reply:
Expanded in 769d98e43